script.js

```diff
@@ -1,4 +1,232 @@
+      " you lose, your score is " +
+      numOfUserWinning +
+      " computer score is " +
+      numOfProgramWinnning
+    );
+  }
+  if (userGuess == "scissors" && programGuess == "stone" && isReversed) {
+    //return userGuess + " VS " + programGuess + " you lose";
+    numOfUserWinning += 1;
+    return (
+      "You chose scissors ✂️." +
+      " VS " +
+      programGuess +
+      " you win, your score is " +
+      numOfUserWinning +
+      " computer score is " +
+      numOfProgramWinnning
+    );
+  }
+  if (userGuess == "paper" && programGuess == "stone" && isReversed) {
+    numOfProgramWinnning += 1;
+    return (
+      userGuess +
+      " VS " +
+      programGuess +
+      " you lose, your score is " +
+      numOfUserWinning +
+      " computer score is " +
+      numOfProgramWinnning
+    );
+  }
+  if (userGuess == "paper" && programGuess == "stone" && isReversed) {
+    numOfProgramWinnning += 1;
+    return (
+      userGuess +
+      " VS " +
+      programGuess +
+      " you lose, your score is " +
+      numOfUserWinning +
+      " computer score is " +
+      numOfProgramWinnning
+    );
+  }
+  if (userGuess == "paper" && programGuess == "scissors" && isReversed) {
+    numOfUserWinning += 1;
+    return (
+      userGuess +
+      " VS " +
+      programGuess +
+      " you win, your score is " +
+      numOfUserWinning +
+      " computer score is " +
+      numOfProgramWinnning
+    );
+  }
+
+  if (userGuess == "paper" && programGuess == "stone" && !isReversed) {
+    numOfUserWinning += 1;
+    return (
+      userGuess +
+      " VS " +
+      programGuess +
+      " you win, your score is " +
+      numOfUserWinning +
+      " computer score is " +
+      numOfProgramWinnning
+    );
+  }
+
+  if (userGuess == "paper" && programGuess == "scissors" && !isReversed) {
+    numOfProgramWinnning += 1;
+    return (
+      userGuess +
+      " VS " +
+      programGuess +
+      " you lose, your score is " +
+      numOfUserWinning +
+      " computer score is " +
+      numOfProgramWinnning
+    );
+  }
+
+  if (userGuess == "stone" && programGuess == "paper" && !isReversed) {
+    numOfProgramWinnning += 1;
+    return (
+      userGuess +
+      " VS " +
+      programGuess +
+      " you lose,your score is " +
+      numOfUserWinning +
+      " computer score is " +
+      numOfProgramWinnning
+    );
+  }
+  if (userGuess == "stone" && programGuess == "paper" && isReversed) {
+    numOfUserWinning += 1;
+    return (
+      userGuess +
+      " VS " +
+      programGuess +
+      " you win,your score is " +
+      numOfUserWinning +
+      " computer score is " +
+      numOfProgramWinnning
+    );
+  }
+  if (userGuess == "stone" && programGuess == "scissors" && !isReversed) {
+    numOfUserWinning += 1;
+
+    return (
+      userGuess +
+      " VS " +
+      programGuess +
+      " you win,your score is " +
+      numOfUserWinning +
+      " computer score is " +
+      numOfProgramWinnning
+    );
+  }
```

Comment:
Remember that you can combine if statements if the outcome is the same. In this case, you can combine both if statements by typing:
if (userGuess == "stone" && programGuess == "scissors" && !isReversed) || (userGuess == "stone" && programGuess == "paper" && isReversed) {
 numOfUserWinning += 1;
    return (
      userGuess +
      " VS " +
      programGuess +
      " you win,your score is " +
      numOfUserWinning +
      " computer score is " +
      numOfProgramWinnning
    );
    }
   
You can actually combine all of your statements into just 3 if statements by adding more statements with the || operator, one where the user wins, one where the user loses and one where the user draws.